esd-group-3/web/WEB-INF/web.xml

```diff
@@ -115,9 +115,8 @@
         <servlet-class>com.ViewReferralsServlet</servlet-class>
     </servlet>
     <servlet>
-        <display-name>DisplayEventsServlet</display-name>
-        <servlet-name>DisplayEventsServlet</servlet-name>
-        <servlet-class>com.DisplayEventsServlet</servlet-class>
+        <servlet-name>UpdateInvoiceServlet</servlet-name>
+        <servlet-class>com.UpdateInvoiceServlet</servlet-class>
```

Comment:
DisplayEventsServlet removed from web.xml but the file still exists and is called in admin_home and client_home?